### PR TITLE
Sm101 create message acceptance tests

### PR DESCRIPTION
**Pull request by AdamHawtin**
Adds acceptance tests for the 'SM101 Internal User To Send Message story' which is now merged into master on Response Operations UI.

Story page:
https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/223281382/SM101+Internal+User+to+Send+Message

### CODE REVIEW

File: acceptance_tests/features/create_message_internal.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Feature: Internal user to send message
+  As an internal ONS user
+  I need to be able to send a secure message
+  So that I can query data that has been received
+
+  Background: User already logged in
+    Given the internal user is already signed in
+    And The internal user has found the associated respondent
+
```
Comment (gemmai95) [inline acceptance_tests/features/create_message_internal.feature]: Can you add tags to each scenario e.g. sm101_s01 and so on

File: acceptance_tests/features/create_message_internal.feature [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  Scenario: 'To' field will be pre-populated with respondent name (up to 100 characters)
+    When they choose to send them a secure message and navigated to the 'send message' page
```
Comment (gemmai95) [inline acceptance_tests/features/create_message_internal.feature]: Can we try to be more explicit than they. Respondent or internal user?

File: acceptance_tests/features/steps/create_message_internal.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import create_message_internal
+
+
+@given("The internal user has found the associated respondent")
+def user_has_found_respondent(_):
+    assert create_message_internal.RESPONDENT_DETAILS.get_respondent_id() is not None
+    assert create_message_internal.RESPONDENT_DETAILS.get_ru_id() is not None
+    assert create_message_internal.RESPONDENT_DETAILS.get_ru_ref() is not None
```
Comment (gemmai95) [inline acceptance_tests/features/steps/create_message_internal.py]: What is the difference between ru_id and ru_ref in this context?

Comment (AdamHawtin) [reply acceptance_tests/features/steps/create_message_internal.py]: RU id's are uuid's used internally in the services, the RU ref is the reference number that the survey teams actually use to identify the RU's. They are 1:1 but both needed for different purposes.

File: acceptance_tests/features/create_message_internal.feature (hunk 2 above)
Comment (AdamHawtin) [reply acceptance_tests/features/create_message_internal.feature]: The wording is copied directly from the story acceptance criteria.

**Pull request merged**